labman/db/tests/test_equipment.py

```diff
@@ -16,26 +17,26 @@
 class TestEquipment(LabmanTestCase):
     def test_list_equipment(self):
         obs = Equipment.list_equipment()
-        # Since we are creating equipment in another test, limit the list to
-        # those that are already in the test DB
-        obs = obs[:18]
-        exp = [{'equipment_id': 1, 'external_id': 'Echo550'},
+        exp = [{'equipment_id': 15, 'external_id': '108379Z'},
+               {'equipment_id': 16, 'external_id': '109375A'},
+               {'equipment_id': 17, 'external_id': '311411B'},
                {'equipment_id': 2, 'external_id': 'BUZZ'},
-               {'equipment_id': 3, 'external_id': 'STINGER'},
-               {'equipment_id': 4, 'external_id': 'PRICKLY'},
-               {'equipment_id': 5, 'external_id': 'LUCY'},
-               {'equipment_id': 6, 'external_id': 'ROB-E'},
-               {'equipment_id': 7, 'external_id': 'RIK-E'},
-               {'equipment_id': 8, 'external_id': 'JER-E'},
-               {'equipment_id': 9, 'external_id': 'HOWE'},
                {'equipment_id': 10, 'external_id': 'Carmen'},
+               {'equipment_id': 1, 'external_id': 'Echo550'},
+               {'equipment_id': 9, 'external_id': 'HOWE'},
+               {'equipment_id': 19, 'external_id': 'IGM-HiSeq4000'},
+               {'equipment_id': 8, 'external_id': 'JER-E'},
                {'equipment_id': 11, 'external_id': 'KF1'},
                {'equipment_id': 12, 'external_id': 'KF2'},
                {'equipment_id': 13, 'external_id': 'KF3'},
                {'equipment_id': 14, 'external_id': 'KF4'},
-               {'equipment_id': 15, 'external_id': '108379Z'},
-               {'equipment_id': 16, 'external_id': '109375A'},
-               {'equipment_id': 17, 'external_id': '311411B'}]
+               {'equipment_id': 18, 'external_id': 'KL-MiSeq'},
```

Comment:
Open question: do we record the chemistry version used (implicitly or explicitly)? If not, should we?

Reply:
@wasade Can you educate me on what the "chemistry version" is?  I am not aware of anything called that being tracked in the db but can't be sure without understanding the concept :)

Reply:
The chemistry on the instruments changes over time and is at least a possible source of technical bias. But Im not sure if that is being tracked. It sounds like it is not. Tagging people on my phone does not seem to be working at the moment, would it be possible to ask Austin for an opinion?